service/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/CromwellBaseResourcesFactory.java

```diff
@@ -274,10 +276,13 @@ public Deployable definition(DefinitionContext definitionContext) {
               .withSystemAssignedManagedServiceIdentity()
               .defineAgentPool(
                   nameGenerator.nextName(ResourceNameGenerator.MAX_AKS_AGENT_POOL_NAME_LENGTH))
-              .withVirtualMachineSize(ContainerServiceVMSizeTypes.STANDARD_A2_V2)
-              .withAgentPoolVirtualMachineCount(1)
-              .withAgentPoolMode(
-                  AgentPoolMode.SYSTEM) // TODO VM Size? Pool Machine count? AgentPoolMode?
+              .withVirtualMachineSize(
```

Comment:
While we're in here, wondering if we could also make autoscaling settings parameterized? (Not sure what they all are offhand.)

Reply:
Agreed, I was also looking at this!